docs/mevblocker/concepts/MEV Concepts/_category_.json

```
@@ -0,0 +1,4 @@
+{
+    "position": 4,
+    "label": "MEV Concepts"
+}
```

```suggestion
}

```
New line for whitespace